ibc-testkit/src/relayer/context.rs

```diff
@@ -356,4 +356,183 @@ where
             signer,
         )
     }
+
+    /// Timeouts a packet from the first context to the second context.
```

Comment:
This language is a bit ambiguous when it comes to delineating whether a packet sent between the two contexts was _**expired**_, or if an actual timeout packet was sent. 

The code seems to indicate the latter, in which case we should say "Sends a timeout packet from the first context to the second context."

Reply:
I agree with you. I updated the comments and the method names.

3abc78e